cocotb/_version.py

```diff
@@ -5,4 +5,4 @@
 # 1) we don't load dependencies by storing it in __init__.py
 # 2) we can import it in setup.py for the same reason
 # 3) we can import it into your module
-__version__ = '1.3.0'
+__version__ = '1.4.0.dev1'
```

Comment:
```suggestion
__version__ = '1.4.0.dev0'
```
maybe?

Reply:
Yes, let's go for "dev0". After reading https://packaging.python.org/guides/distributing-packages-using-setuptools/#standards-compliance-for-interoperability I got the impression that numbers start with 1, but https://www.python.org/dev/peps/pep-0440/ is quite clear that 0 is allowed.